src/pretalx/locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -6291,8 +6291,10 @@ msgstr "Passwortwiederherstellung"
 #: pretalx/schedule/forms.py:21
 msgid "Please click and drag to mark the availability during the conference."
 msgstr ""
-"Mit klicken und ziehen kannst du die zur Verfügung stehenden Zeiten "
-"markieren."
+"Mit Klicken und Ziehen kannst Zeiträume markieren, in denen du verfügbar bist. (= grün)"
```

Comment:
Hmm, das ist auf jeden Fall eine bessere Erklärung, aber es wäre gut, die auch im Original zu haben (dann kommen auch andere Sprachen in den Genuss, und englische User).